compliance-monitor/requirements.in

```diff
@@ -0,0 +1,9 @@
+argon2_cffi
+bcrypt
+fastapi
+jinja2
+passlib
+psycopg2
+PyYAML
+ruamel.yaml
```

Comment:
I had the impression that ruamel.yaml was superior (because it can do roundtrips etc.), but the question remains why I have both in there. I will check that!